src/tools/rbd/action/DiskUsage.cc

```diff
@@ -102,6 +111,7 @@ static int do_disk_usage(librbd::RBD &rbd, librados::IoCtx &io_ctx,
     f->open_array_section("images");
   } else {
     tbl.define_column("NAME", TextTable::LEFT, TextTable::LEFT);
+    tbl.define_column("ID", TextTable::LEFT, TextTable::LEFT);
```

Comment:
What's the use-case here for showing the image id? Can this just be restricted to the json/xml formated output to avoid cluttering the output?

Reply:
Yeah, it's more reasonable to restricted to the json/xml formated output, and we use `rbd du --format json` in actual application. Thx.